README.md

```diff
@@ -24,10 +24,21 @@ npm install @react-native-community/art --save
 
 ### Mostly automatic linking
 
+If using React Native >= 0.60, add the React-ART pod to your Podfile in ios folder:
+
 ```js
-react-native link @react-native-community/art
+pod 'ReactNativeART', :path => '../node_modules/@react-native-community/art'
```

Comment:
This is not true. In react-native >= 0.60 this dependency should be auto-linked.